conf/application.conf

```diff
@@ -109,6 +109,11 @@ microservice {
       host = localhost
       port = 7775
     }
+
```

Comment:
If we just want the acceptance/performance tests to work we are fine with using the bta.url property. However, this isn't doing any harm.